source/_integrations/hisense_ac_plugin.markdown

```diff
@@ -0,0 +1,197 @@
+2. Click **Create Credential**
+3. Select "Hisense Air Conditioner"
+4. Complete the OAuth2 authorization process
+5. Use the created credential when configuring the integration
+
+## Entities
+
+### Climate Entities
+- **Main Control**: Power, mode, temperature, fan speed
+- **Attributes**: Indoor temperature, indoor humidity, energy consumption
+
+### Sensor Entities
+- **Temperature Sensors**: Indoor temperature, inlet water temperature, outlet water temperature
+- **Humidity Sensors**: Indoor humidity
+- **Energy Sensors**: Power consumption statistics
+- **Fault Sensors**: Device fault status
+
+### Switch Entities
+- **Feature Switches**: Quiet mode, rapid mode, 8-degree heating
+- **Status Switches**: Device online status
+
+### Number Entities
+- **Temperature Settings**: Target temperature adjustment
+- **Humidity Settings**: Target humidity adjustment
+
+## Troubleshooting
+
+### Common Issues
+
+#### Devices Not Appearing
+- Verify OAuth2 authorization was successful
+- Check that devices are connected to ConnectLife platform
+- Ensure network connectivity
+- Check Home Assistant logs for error messages
+
+#### Controls Not Responding
+- Verify device online status
+- Check network connection
+- Restart Home Assistant
+- Re-authorize OAuth2 if needed
+
+#### Delayed Status Updates
+- Check WebSocket connection status
+- Verify network connectivity
+- Restart the integration
+
+### Logs
+
+Enable debug logging to troubleshoot issues:
+
+```yaml
+logger:
+  logs:
+    custom_components.hisense_ac_plugin: debug
+```
+
+### Diagnostics
+
+Export diagnostic information:
+
+1. Go to **Settings** → **System** → **Logs**
+2. Click **Download Diagnostics**
+3. Select "Hisense AC Plugin" integration
+4. Download the diagnostic file
+
+## Automation Examples
+
+### Temperature Control
+```yaml
+automation:
+  - alias: "Auto AC Control"
+    trigger:
+      - platform: numeric_state
+        entity_id: sensor.living_room_temperature
+        above: 26
+    action:
+      - service: climate.set_temperature
+        target:
+          entity_id: climate.living_room_ac
+        data:
+          temperature: 24
+          hvac_mode: cool
+```
+
+### Energy Monitoring
+{% raw %}
+```yaml
+automation:
+  - alias: "High Energy Alert"
+    trigger:
+      - platform: numeric_state
+        entity_id: sensor.ac_power_consumption
+        above: 2.0
+    action:
+      - service: notify.mobile_app
+        data:
+          message: "AC energy consumption high: {{ states('sensor.ac_power_consumption') }}kWh"
+```
+{% endraw %}
+
+### Fault Notification
+{% raw %}
+```yaml
+automation:
+  - alias: "Device Fault Alert"
+    trigger:
+      - platform: state
+        entity_id: sensor.ac_fault_status
+        to: "fault"
+    action:
+      - service: notify.mobile_app
+        data:
+          message: "AC device fault: {{ state_attr('sensor.ac_fault_status', 'fault_details') }}"
+```
+{% endraw %}
+
+## Support
```

Comment:
2. We don't usually include information like the licensing and GH support. For more information, refer to the documentation [Integration documentation structure](https://developers.home-assistant.io/docs/documenting/integration-docs-examples) and [examples](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown)